inc/modules/export/namespace.php

```diff
@@ -164,7 +158,6 @@ function filetypes() {
 			'pdf' => '.pdf',
 			'print_pdf' => '._print.pdf',
 			'mobi' => '.mobi',
-			'icml' => '.icml',
```

Comment:
We need to leave this mapping in place so that the export menu displays properly in case a user previously made an ICML export long ago and has one in its export list. Same logic for leaving mobi here ...

Reply:
I see, makes sense. I'll revert this line.